client/js/components/procedure/DpSimplifiedNewStatementForm.vue

```diff
@@ -580,6 +581,11 @@ export default {
   },
 
   methods: {
+    abort () {
+      const href = `${Routing.generate('DemosPlan_procedure_import', { procedureId: this.procedureId })}/#import#StatementPdfImport`
```

Comment:
Is the '#' needed before the 'import#StatementPdfImport'?

Reply:
Yes this is needed since we want to redirect directly to the import section of the import center after aborting the confirmation of a pdf import.